spec/helpers/reminder_helper_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe ReminderHelper do
+  name1 = 'name1'
+  name2 = 'name2'
+  reminder_names = [
+    name1, name2
+  ]
+
+  describe '#reminders?' do
+    it 'returns false when data has no active_reminder method' do
+      expect(reminders?({})).to be_falsey
+    end
+
+    it 'returns false when data has no active reminder' do
+      user = FactoryBot.create(:user)      
```

Comment:
You could move up the creation of a user into a let

Reply:
moved